index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+</head>
+<body>
+    <header class="header">
+            <a href="#" class="logo">My Logo</a>
+            <nav>
+                <i class="material-symbols-outlined menu_icon">
+                    menu
+                </i>
```

Comment:
- [ ] Good job making most of the changes requested by the previous reviewer. however, the margin from the top to the start of your logo and nav bar hamburger needs a little adjustment. Find below the comparison view for your reviewer

## Your view:
![image](https://user-images.githubusercontent.com/50990925/219551013-89b91369-f928-409a-82b3-a7a0bdad0fb5.png)

## figma:
![image](https://user-images.githubusercontent.com/50990925/219551241-09f0832b-c17c-423f-b9e1-5cd3d9df8140.png)